internal/serve/graphql/schema/statechange.graphqls

```diff
@@ -0,0 +1,38 @@
+# GraphQL StateChange type - represents changes to blockchain state
+# This type has many nullable fields to handle various state change scenarios
+# TODO: Break state change type into interface design and add sub types that implement the interface
+type StateChange{
+  id:                         String!
+  accountId:                  String!           
+  stateChangeCategory:        StateChangeCategory!
+  stateChangeReason:          StateChangeReason
+  ingestedAt:                 Time!
+  ledgerCreatedAt:            Time!
+  ledgerNumber:               UInt32!           
+  
+  # GraphQL Nullable fields - these map to sql.NullString in Go
+  # GraphQL handles nullable fields gracefully - they return null if not set
+  tokenId:                    String              
+  amount:                     String
+  claimableBalanceId:         String   
+  liquidityPoolId:            String      
+  offerId:                    String              
+  signerAccountId:            String      
+  spenderAccountId:           String     
+  sponsoredAccountId:         String   
+  sponsorAccountId:           String     
+  
+  # GraphQL fields for JSONB data - require custom resolvers
+  # These fields need special handling to convert between Go types and GraphQL
+  signerWeights:              String        
+  thresholds:                 String           
+  flags:                      [String!]             
+  keyValue:                   String             
```

Comment:
Should we create types for these rather than treating them as serialized json? If so we can do it in a separate PR.

Reply:
@JakeUrban Yes thats the plan for the next PR - adding types and pagination